<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #6667ab;</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>451</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>431</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #6667ab</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>h</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_b">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_a">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>16</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>'</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_m">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>b</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_h">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>f</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_hf">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>b</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>h</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>f</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_bf">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;</string>
      </property>
      <property name="text">
       <string>Класс бетона</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>410</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;</string>
      </property>
      <property name="text">
       <string>Класс арматуры</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>530</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>ММ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>ММ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>ММ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>ММ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>230</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>ММ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>280</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>кH*м</string>
      </property>
     </widget>
     <widget class="QLabel" name="output_as">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>510</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #fff;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="output_bet_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #fff;</string>
      </property>
      <item>
       <property name="text">
        <string>B25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B35</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B40</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>520</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>ММ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>520</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_as">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>510</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #6667ab;
background-color: #fff;</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QComboBox" name="output_bet_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #fff;</string>
      </property>
      <item>
       <property name="text">
        <string>A400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A240</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A300</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A500</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>871</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="text">
     <string>КАЛЬКУЛЯТОР ДЛЯ РАСЧЕТА ПЛОЩАДИ СЕЧЕНИЯ АРМАТУРЫ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>821</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>dOW005Q-0601.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="odin">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>660</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #6667ab;
background-color: #fff;</string>
    </property>
    <property name="text">
     <string>сохранить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="text">
     <string>КАЛЬКУЛЯТОР</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_user">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>580</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calcl">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>580</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #6667ab;
background-color: #fff;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_otv">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>580</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1296</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
